CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
```

Comment:
Consider specifying the schema and table name using `.ToTable("Attendances")` if your conventions don't do it automatically. This will help avoid EF Core creating a pluralized or undesired table name depending on conventions.

**Suggested change**
```suggestion
builder.ToTable("Attendances");
```